### ISSUE TITLE
non-informative exception message is thrown when prefix/namespace is missing

### ISSUE THREAD

**Issue by ymark** [isl/x3ml#7]
If we use a resource from a namespace that have not been declared in the namespace section of the mappings file (i.e. produce an rdfs:label, however the rdfs namespace is not declared in the corresponding section) an exception message is thrown however it is not informative at all.

It reports errors with fully qualified URIs in the output model (jena)

**Comment by ymark** [comment]
To check also if the same non-informative message appears when using custom prefixes/namespaces  not defined in the namespace section (i.e. CRM)

**Comment by ymark** [comment]
Confirmed that the same error occurs when using prefixes from custom namespaces. 
A snapshot of the exception message is the following:
```
Exception in thread "main" com.hp.hpl.jena.shared.BadURIException: Only well-formed absolute URIrefs can be included in RDF/XML output: <n> Code: 57/REQUIRED_COMPONENT_MISSING in SCHEME: A component that is required by the scheme is missing.
	at com.hp.hpl.jena.xmloutput.impl.BaseXMLWriter.checkURI(BaseXMLWriter.java:820)
	at com.hp.hpl.jena.xmloutput.impl.BaseXMLWriter.xmlnsDecl(BaseXMLWriter.java:339)
	at com.hp.hpl.jena.xmloutput.impl.Unparser.printNameSpaceDefn(Unparser.java:1073)
	at com.hp.hpl.jena.xmloutput.impl.Unparser.wRDF(Unparser.java:338)
	at com.hp.hpl.jena.xmloutput.impl.Unparser.write(Unparser.java:247)
	at com.hp.hpl.jena.xmloutput.impl.Abbreviated.writeBody(Abbreviated.java:142)
	at com.hp.hpl.jena.xmloutput.impl.BaseXMLWriter.writeXMLBody(BaseXMLWriter.java:492)
	at com.hp.hpl.jena.xmloutput.impl.BaseXMLWriter.write(BaseXMLWriter.java:464)
	at com.hp.hpl.jena.xmloutput.impl.Abbreviated.write(Abbreviated.java:127)
	at com.hp.hpl.jena.xmloutput.impl.BaseXMLWriter.write(BaseXMLWriter.java:450)
	at com.hp.hpl.jena.rdf.model.impl.ModelCom.write(ModelCom.java:354)
	at eu.delving.x3ml.engine.ModelOutput.writeXML(ModelOutput.java:101)
```

**Comment by ymark** [comment]
Resolved with commit fe7eb29d
